javascript/javascript1/week2/student-maneger.js

```diff
@@ -1,19 +1,25 @@
-let class14Students = ['maha', 'Elias', 'Anil' , 'Deepti'];
-function addStudentToClass(studentName) { 
-     if (class14Students.length >= 6) {
-       return 'no more student can be added to class 14';
-    } else if  (class14Students.includes(studentName)) {
-       return 'Student ' + studentName +  ' is already in the class'; 
-    } else if  (class14Students.length < 6 || studentName == 'queen' ) {
-        return  studentName + ' is added to class 14 as the '+
-         class14Students.push(studentName)+ 'th ' + 'student';
-    }  
+let class07Students = ['Maha', 'kamal', 'Gary', 'Karolina', 'shruthi'];
```

Comment:
The key word for the variable can still be "const" instead of "let", as you do not re-assign anything to the variable but push the studentName in to class07students array.

reference : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

look for "const in objects and arrays".

And in this exercise your array should be empty.